github/project.go

```diff
@@ -92,13 +92,13 @@ func rawHost(host string) string {
 	}
 }
 
-func useHttpProtocol() bool {
-	https := os.Getenv("HUB_PROTOCOL")
-	if https == "" {
-		https, _ = git.Config("hub.protocol")
+func useProtocol(proto string) bool {
```

Comment:
Let's call this method `preferredProtocol()`. Then its usage would be:

```go
if "https" == preferredProtocol() {
  # ...
}
```

Simplicity.